src/ResourceManagement/EventHub/Microsoft.Azure.Management.EventHub/project.json

```diff
@@ -0,0 +1,57 @@
+{
+  "version": "1.0.1-preview",
```

Comment:
I would suggest starting with 0, until you're comfortable that there won't be breaking changes.